lib/models/device.coffee

```diff
@@ -707,6 +707,8 @@ exports.restart = exports.restartApplication
 # @memberof resin.models.device
 #
 # @param {String} uuid - device uuid
+# @param {Object} options - options (optional)
```

Comment:
nit: you can mark optional arguments using brackets like `[options] - options` in JSDoc